unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/SceneController.cs

```diff
@@ -77,9 +77,6 @@ public void Start()
             {
                 PoolManagerFactory.EnsureEntityPool(prewarmEntitiesPool);
             }
-
-            // Warmup some shader variants
-            Resources.Load<ShaderVariantCollection>("ShaderVariantCollections/shaderVariants-selected").WarmUp();
         }
 
         public void Dispose()
```

Comment:
For correctness don't forget to unsubscribe to the event on the `Dispose()` method.